docs/sources/mimir/references/http-api/index.md

```diff
@@ -578,14 +591,9 @@ This method of counting is most useful for understanding what label values are r
 Two subsequent calls will likely return similar results, because this window of time is not related to the block cutting on ingesters.
 Values will change only as a result of changes in the data ingested by Mimir.
 
-The items in the field `labels` are sorted by `series_count` in DESC order and by `label_name` in ASC order.
-The items in the field `cardinality` are sorted by `series_count` in DESC order and by `label_value` in ASC order.
+#### Caching
 
-The count of `cardinality` items is limited by request param `limit`.
-
-This endpoint is disabled by default and can be enabled via the `-querier.cardinality-analysis-enabled` CLI flag (or its respective YAML config option).
-
-Requires [authentication](#authentication).
+The response of this API endpoint could be fetched from the query-frontend results cache if `-query-frontend.cache-results` is enabled and `-query-frontend.results-cache-ttl-for-cardinality-query` set to a value greater than 0.
```

Comment:
```suggestion
The response of this API endpoint can be fetched from the query-frontend results cache if `-query-frontend.cache-results` is enabled and `-query-frontend.results-cache-ttl-for-cardinality-query` set to a value greater than `0`.
```
See previous comment about passive voice.

Reply:
How does this sound?

```suggestion
The query-frontend can return a stale response fetched from the query results cache if `-query-frontend.cache-results` is enabled and `-query-frontend.results-cache-ttl-for-cardinality-query` set to a value greater than `0`.
```